specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-preview/subscriptions.json

```diff
@@ -868,6 +1088,179 @@
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/TenantPolicy",
+          "description": "Tenant policy properties."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "GetTenantPolicyListResponse": {
+      "description": "Tenant policy information list.",
+      "properties": {
+        "value": {
+          "description": "The list of tenant policies.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/GetTenantPolicyResponse"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "BillingAccountPoliciesResponse": {
+      "description": "Billing account policies information.",
+      "properties": {
+        "id": {
+          "description": "Fully qualified ID for the policy.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "name": {
+          "description": "Policy name.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type, Microsoft.Subscription/policies.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/BillingAccountPoliciesResponseProperties",
+          "description": "Put Alias response properties."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "BillingAccountPoliciesResponseProperties": {
+      "description": "Put billing account policies response properties.",
+      "properties": {
+        "serviceTenants": {
+          "description": "Service tenant for the billing account.",
+          "type": "array",
+          "items":{
+            "type": "object",
+            "$ref": "#/definitions/ServiceTenantResponse",
+            "description": "Billing account service tenant."
+          }
+        },
+        "allowTransfers": {
+          "description": "Determine if the transfers are allowed for the billing account",
+          "type": "boolean"
+        }
+      }
+    },
+    "ServiceTenantResponse": {
+      "description": "Billing account service tenant.",
+      "properties": {
+        "tenantId": {
+          "description": "Service tenant id.",
+          "type": "string"
+        },
+        "tenantName": {
+          "description": "Service tenant name.",
+          "type": "string"
+        }
+      }
+    },
+    "RedeemSubscriptionRequest": {
+      "description": "The parameters required to redeem a new subscription.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/RedeemSubscriptionRequestProperties",
+          "description": "Put alias request properties."
```

Comment:
Update description. Please cross check the descriptions in other places too.

Reply:
Sure. Updated